controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -2017,6 +2020,66 @@ private HttpResponse getPrivateServiceInfo(String tenantName, String application
         return new SlimeJsonResponse(slime);
     }
 
+    private HttpResponse dropDocumentsStatus(String tenant, String application, String instance, String environment, String region, Optional<ClusterSpec.Id> clusterId) {
+        ZoneId zone = ZoneId.from(environment, region);
+        if (!zone.environment().isManuallyDeployed())
+            throw new IllegalArgumentException("Drop documents status is only available for manually deployed environments");
+
+        ApplicationId applicationId = ApplicationId.from(tenant, application, instance);
+        NodeFilter filters = NodeFilter.all()
+                .states(Node.State.active)
+                .applications(applicationId)
+                .clusterTypes(Node.ClusterType.content, Node.ClusterType.combined);
+        List<Node> nodes = controller.serviceRegistry().configServer().nodeRepository().list(zone, clusterId.map(filters::clusterIds).orElse(filters));
+        if (nodes.isEmpty()) {
+            throw new NotExistsException("No content nodes found for %s%s in %s".formatted(
+                    applicationId.toFullString(), clusterId.map(id -> " cluster " + id).orElse(""), zone));
+        }
+
+        Instant readiedAt = null;
+        int numNoReport = 0, numInitial = 0, numDropped = 0, numReadied = 0, numStarted = 0;
+        for (Node node : nodes) {
+            Inspector report = Optional.ofNullable(node.reports().get("dropDocuments"))
+                    .map(json -> SlimeUtils.jsonToSlime(json).get()).orElse(null);
+            if (report == null) numNoReport++;
+            else if (report.field("startedAt").valid()) {
+                numStarted++;
+                readiedAt = SlimeUtils.instant(report.field("readiedAt"));
+            } else if (report.field("readiedAt").valid()) numReadied++;
+            else if (report.field("droppedAt").valid()) numDropped++;
+            else numInitial++;
+        }
+
+        if ((numInitial > 0 && numNoReport > 0) ||
+            (numReadied > 0 && (numNoReport > 0 || numInitial > 0 || numDropped > 0)) ||
+            (numStarted > 0 && (numInitial > 0 || numDropped > 0)))
+            return ErrorResponse.conflict("Inconsistent state, try restarting drop documents again");
```

Comment:
"Inconsistent state" may cause the reader to think the Vespa team messed up, whereas this may occur naturally through topology changes.  Rephrase?

Reply:
"Last dropping of documents may have failed to clear all documents due to concurrent topology changes, consider retrying."